ruby.mustache

```diff
@@ -57,9 +57,9 @@ def send_request
   req =  Net::HTTP::{{{method}}}{{#body.has_multipart_body}}::Multipart{{/body.has_multipart_body}}.new(uri{{#body.has_multipart_body}}, body{{/body.has_multipart_body}})
   {{! ----- }}
   {{#headers.has_headers}}
-  {{#headers.header_list}}
```

Comment:
Moved this down so we only output the comment once for the entire list of headers.